package.json

```diff
@@ -4,6 +4,7 @@
   "description": "etherlime is an ethereum development and deployment framework based on ethers.js",
   "main": "index.js",
   "scripts": {
+    "postinstall" : "node --eval 'console.log(\"By installing Etherlime you are automatically opt-in for analytics. To disable it run `etherlime opt-out` cli command.\")'",
```

Comment:
By installing Etherlime you automatically opt-in for analytics. To opt-out run `etherlime opt-out`.